assets/section-featured-product.css

```diff
@@ -0,0 +1,57 @@
+.featured-product .product__media-list {
```

Comment:
I think there should be more margin at the bottom when it's the last section, otherwise it gets pretty close to the footer: [screenshot](https://screenshot.click/21-08-48i5a-o3til.png)

Reply:
+1

Reply:
Fixed :D